docs/basics/download_nwb.ipynb

```diff
@@ -61,8 +61,10 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    download_loc = "." 
</pre><p>haha my last feedback must have been confusing.</p><p>The 'relative filepath' comment would go above the line with the filepath (download_loc...)</p><p>The comment about the dandiset id was good info. If you want to include instructions on that it would go well in the markdown, right after where it says "use its ID in dandiset_id."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>